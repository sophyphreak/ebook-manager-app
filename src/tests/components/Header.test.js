import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header/Header';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header />);
});

test('should render Header correctly', () => {
  expect(wrapper).toMatchSnapshot();
});