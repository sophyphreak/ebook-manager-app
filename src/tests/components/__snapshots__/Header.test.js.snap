// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavbarToggler
          className="d-lg-none"
          onClick={[Function]}
          tag="button"
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </NavbarToggler>,
        <NavbarBrand
          href="#"
          tag="a"
        />,
        <NavbarToggler
          className="d-md-down-none mr-auto"
          onClick={[Function]}
          tag="button"
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </NavbarToggler>,
        <NavbarToggler
          className="d-md-down-none"
          onClick={[Function]}
          tag="button"
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </NavbarToggler>,
      ],
      "className": "app-header navbar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <span
            className="navbar-toggler-icon"
          />,
          "className": "d-lg-none",
          "onClick": [Function],
          "tag": "button",
          "type": "button",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "navbar-toggler-icon",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "href": "#",
          "tag": "a",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <span
            className="navbar-toggler-icon"
          />,
          "className": "d-md-down-none mr-auto",
          "onClick": [Function],
          "tag": "button",
          "type": "button",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "navbar-toggler-icon",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <span
            className="navbar-toggler-icon"
          />,
          "className": "d-md-down-none",
          "onClick": [Function],
          "tag": "button",
          "type": "button",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "navbar-toggler-icon",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        "type": [Function],
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavbarToggler
            className="d-lg-none"
            onClick={[Function]}
            tag="button"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </NavbarToggler>,
          <NavbarBrand
            href="#"
            tag="a"
          />,
          <NavbarToggler
            className="d-md-down-none mr-auto"
            onClick={[Function]}
            tag="button"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </NavbarToggler>,
          <NavbarToggler
            className="d-md-down-none"
            onClick={[Function]}
            tag="button"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </NavbarToggler>,
        ],
        "className": "app-header navbar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <span
              className="navbar-toggler-icon"
            />,
            "className": "d-lg-none",
            "onClick": [Function],
            "tag": "button",
            "type": "button",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "navbar-toggler-icon",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "href": "#",
            "tag": "a",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <span
              className="navbar-toggler-icon"
            />,
            "className": "d-md-down-none mr-auto",
            "onClick": [Function],
            "tag": "button",
            "type": "button",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "navbar-toggler-icon",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <span
              className="navbar-toggler-icon"
            />,
            "className": "d-md-down-none",
            "onClick": [Function],
            "tag": "button",
            "type": "button",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "navbar-toggler-icon",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          "type": [Function],
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
